splinterd/src/main.rs

```diff
@@ -665,6 +670,39 @@ fn start_daemon(matches: ArgMatches, _log_handle: Handle) -> Result<(), UserErro
     #[cfg(feature = "log-config")]
     {
         let appenders = if let Some(appenders) = config.appenders() {
+            let check_file_readability = |path: &Path| {
```

Comment:
Perhaps we could move all of the logging setup into its own module in a future PR.